src/sentry/static/sentry/app/components/avatar/baseAvatar.jsx

```diff
@@ -67,32 +77,16 @@ class BaseAvatar extends React.Component {
     return remoteImageSize || allowed || DEFAULT_GRAVATAR_SIZE;
   };
 
-  buildGravatarUrl = () => {
-    let {gravatarId} = this.props;
-    let url = ConfigStore.getConfig().gravatarBaseUrl + '/avatar/';
-
-    url += MD5(gravatarId);
-
-    let query = {
-      s: this.getRemoteImageSize() || undefined,
-      d: this.props.default || 'blank',
-    };
-
-    url += '?' + qs.stringify(query);
-
-    return url;
-  };
-
   buildUploadUrl = () => {
     let {uploadPath, uploadId} = this.props;
 
     return `/${uploadPath || 'avatar'}/${uploadId}/?${qs.stringify({
-      s: 120,
+      s: DEFAULT_REMOTE_SIZE,
     })}`;
   };
 
   handleLoad = () => {
-    this.setState({showBackupAvatar: true});
+    this.setState({showBackupAvatar: false});
```

Comment:
If remote image gets loaded, we should remove the backup avatar. Otherwise transparent avatars would show the backup underneath.

A casualty of this is having gravatars be the default avatar when there is an email, but I think letter avatars are pretty good as the new default.